### ISSUE TITLE
Resources$NotFoundException: File res/drawable-hdpi-v4/error_image.png

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#571]
```
E/AndroidRuntime: FATAL EXCEPTION: main
   Process: openfoodfacts.github.scrachx.openfood, PID: 12295
   android.content.res.Resources$NotFoundException: File res/drawable-hdpi-v4/error_image.png from xml type xml resource ID #0x7f020064
 at android.content.res.Resources.loadXmlResourceParser(Resources.java:2899)
 at android.content.res.Resources.loadXmlResourceParser(Resources.java:2854)
 at android.content.res.Resources.getXml(Resources.java:1236)
 at android.support.graphics.drawable.VectorDrawableCompat.create(VectorDrawableCompat.java:586)
 at openfoodfacts.github.scrachx.openfood.views.adapters.NutrientLevelListAdapter.getView(NutrientLevelListAdapter.java:60)
 at android.widget.AbsListView.obtainView(AbsListView.java:2346)
 at android.widget.ListView.onMeasure(ListView.java:1158)
 at android.view.View.measure(View.java:18794)
 at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
 at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1465)
 at android.widget.LinearLayout.measureVertical(LinearLayout.java:748)
 at android.widget.LinearLayout.onMeasure(LinearLayout.java:630)
 at android.view.View.measure(View.java:18794)
 at android.support.v4.widget.NestedScrollView.measureChildWithMargins(NestedScrollView.java:1420)
 at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
 at android.support.v4.widget.NestedScrollView.onMeasure(NestedScrollView.java:482)
 at android.view.View.measure(View.java:18794)
 at android.support.v4.view.ViewPager.onMeasure(ViewPager.java:1658)
 at android.view.View.measure(View.java:18794)
 at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
 at android.support.design.widget.CoordinatorLayout.onMeasureChild(CoordinatorLayout.java:714)
 at android.support.design.widget.HeaderScrollingViewBehavior.onMeasureChild(HeaderScrollingViewBehavior.java:90)
 at android.support.design.widget.AppBarLayout$ScrollingViewBehavior.onMeasureChild(AppBarLayout.java:1375)
 at android.support.design.widget.CoordinatorLayout.onMeasure(CoordinatorLayout.java:784)
 at android.view.View.measure(View.java:18794)
 at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
 at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
 at android.support.v7.widget.ContentFrameLayout.onMeasure(ContentFrameLayout.java:139)
 at android.view.View.measure(View.java:18794)
 at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
 at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1465)
 at android.widget.LinearLayout.measureVertical(LinearLayout.java:748)
 at android.widget.LinearLayout.onMeasure(LinearLayout.java:630)
 at android.view.View.measure(View.java:18794)
 at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
 at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
 at android.view.View.measure(View.java:18794)
 at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
 at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1465)
 at android.widget.LinearLayout.measureVertical(LinearLayout.java:748)
 at android.widget.LinearLayout.onMeasure(LinearLayout.java:630)
 at android.view.View.measure(View.java:18794)
 at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5951)
 at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
 at com.android.internal.policy.PhoneWindow$DecorView.onMeasure(PhoneWindow.java:2643)
 at android.view.View.measure(View.java:18794)
 at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:2100)
 at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1216)
 at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1452)
 at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1107)
 at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6013)
 at android.view.Choreographer$CallbackRecord.run(Choreographer.java:858)
 at android.view.Choreographer.doCallbacks(Choreographer.java:670)
 at android.view.Choreographer.doFrame(Choreographer.java:606)
at android.view 
```

**Comment by teolemon** [comment]
not quite sure why ```drawable-hdpi-v4/error_image.png```is called (error_image exists)

**Comment by teolemon** [comment]
http://stackoverflow.com/questions/12852702/android-support-multiple-screen

**Comment by teolemon** [comment]
```Error:Execution failed for task ':app:mergeOffDebugResources'.
> [drawable-hdpi-v4/error_image] /Users/pierreslam/Desktop/cloneandroid/openfoodfacts-androidapp/app/src/main/res/drawable-hdpi/error_image.png	[drawable-hdpi-v4/error_image] /Users/pierreslam/Desktop/cloneandroid/openfoodfacts-androidapp/app/src/main/res/drawable-hdpi-v4/error_image.png: Error: Duplicate resources```

**Comment by teolemon** [comment]
While trying to create a folder called drawable-hdpi-v4/error_image.png

**Comment by teolemon** [comment]
Ok. It seems that this bug is in the Live version (at least on F-Droid).

**Comment by jaztriumph** [comment]
@teolemon  I think it is a compatibility problem which occurs on android versions below 21 and I can resolve all these issues once #693 is merged.The above error occurs because in  [views/adapters/NutrientLevelListAdapter.java#L60](https://github.com/openfoodfacts/openfoodfacts-androidapp/blob/a80c79142a4f441d8c3e6f87ea29f32567829d62/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/adapters/NutrientLevelListAdapter.java#L60)  `VectorDrawableCompat.create()` cannot load png images it will only inflate vector drawables, so when png is encountered it throughs an error. 

This can be resolved using `AppCompatDrawable.getDrawable()` . For more info [see here](https://stackoverflow.com/questions/40241018/appcompatdrawablemanager-get-vs-vectordrawablecompat-create).

**Comment by Shubham2320** [comment]
I think this can also be solved in this way PR #784.

**Comment by jaztriumph** [comment]
@Shubham2320 This issue is not in just one file PR #783 fixes all those as the core reason for all of them is same.
